homeassistant/components/neato/__init__.py

```diff
@@ -173,7 +174,7 @@
 }
 
 
-def setup(hass, config):
+async def async_setup(hass, config):
```

Comment:
Please revert this change. This integration isn't implemented to support asyncio.

Reply:
Done